scripts/ci/rust_checks.py

```diff
@@ -33,7 +33,7 @@ def run_cargo(cargo_cmd, cargo_args: str, clippy_conf: str | None = None) -> Tim
     env["RUSTFLAGS"] = "--deny warnings"
     env["RUSTDOCFLAGS"] = "--deny warnings --deny rustdoc::missing_crate_level_docs"
     if clippy_conf is not None:
-        env["CLIPPY_CONF_DIR"] = clippy_conf
+        env["CLIPPY_CONF_DIR"] = f"{os.getcwd()}/{clippy_conf}"
```

Comment:
should point out that this was necessary for ci to succeed if this is the solution now